package.json

```diff
@@ -133,7 +133,7 @@
     "build:watch": "watch 'yarn build' src"
   },
   "engines": {
-    "node": ">=14.18.0"
+    "node": ">=16.0.0"
```

Comment:
Note: Update here to node `>=16.0.0` instead of just Node v15 due to LTS support of node versions